src/app/wallet/wallet/send/send-confirmation-modal/send-confirmation-modal.component.scss

```diff
@@ -0,0 +1,101 @@
+@import "./src/assets/_config"; // import shared colors etc.
+
+
+// ------ UI ------ //
+
+.title {
+  // FIXME: swith to @extend %box-title;
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 13px;
+}
+
+.help-text {
+  // FIXME: switch to: @extend %help-text;
+  @extend %disable-select;
+  color: $text-muted;
+  font-size: 11px;
+  margin-bottom: 25px;
+  cursor: help;
+}
+
+
+// ------ LAYOUT ------ //
+
+.dialog-content {
+  width: 25rem;
+  padding: 20px 15px 10px;
+  text-align: center;
+}
+
+.tx-info { // info about public/blind/anon TX
+  @extend %disable-select;
+  .icon {
+    width: auto;
+    height: auto;
+    font-size: 32px;
+    color: $color;
+  }
+  .help-text {
+    margin: 5px auto 20px;
+    max-width: 20rem;
+  }
+}
+
+.tx-amount {
+  & > .big {
+    font-size: 20px;
+  }
+  & > .small {
+    font-size: 14px;
+  }
+  & > .point {
+    font-weight: bold;
+    margin: 0 2px;
+  }
+  & > .currency {
+    font-size: 17px;
+  }
+  & > .fiat {
```

Comment:
Can we hide the fiat div via css rather than comments?
Easier later on & cleaner.